js/admin/ps_emailalerts.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * 2007-2022 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2022 PrestaShop SA
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+$(document).on('ready', function() {
+  let $ma_merchant_order_radio_off = $('#MA_MERCHANT_ORDER_off');
+  let $ma_merchant_oos_radio_off = $('#MA_MERCHANT_OOS_off');
+  let $ma_merchant_return_slip_radio_off = $('#MA_RETURN_SLIP_off');
+
+  let $order_emails_form_group = $('#MA_MERCHANT_ORDER_EMAILS').parents('.form-group');
+  let $oos_emails_form_group = $('#MA_MERCHANT_OOS_EMAILS').parents('.form-group');
+  let $return_slip_emails_form_group = $('#MA_RETURN_SLIP_EMAILS').parents('.form-group');
+
+  // Check if che options are disabled (hide e-mails text input)
+  if($ma_merchant_order_radio_off.prop('checked')){
+    $order_emails_form_group.hide();
+  }
+  if($ma_merchant_oos_radio_off.prop('checked')){
+    $oos_emails_form_group.hide();
+  }
+  if($ma_merchant_return_slip_radio_off.prop('checked')){
+    $return_slip_emails_form_group.hide();
+  }
+
+  // Bind change event
+  $(document).on('change', 'input[id^="MA_MERCHANT_ORDER"]', function(){
+    $order_emails_form_group.toggle($('#MA_MERCHANT_ORDER_on').is(':checked'));
+  });
+
+  $(document).on('change', 'input[id^="MA_MERCHANT_OOS"]', function(){
+    $oos_emails_form_group.toggle($('#MA_MERCHANT_OOS_on').is(':checked'));
+  });
+
+  $(document).on('change', 'input[id^="MA_RETURN_SLIP"]', function(){
+    $return_slip_emails_form_group.toggle($('#MA_RETURN_SLIP_on').is(':checked'));
+  });
```

Comment:
I'm not a big fan of using `input[id^="MA_RETURN_SLIP"]`, I think it's not ideal regarding performance.
What about something like this?:

```suggestion
  let $ma_merchant_order_radio = $('#MA_MERCHANT_ORDER_off, #MA_MERCHANT_ORDER_on');
  let $ma_merchant_oos_radio = $('#MA_MERCHANT_OOS_off, #MA_MERCHANT_OOS_on');
  let $ma_merchant_return_slip_radio = $('#MA_RETURN_SLIP_off, #MA_RETURN_SLIP_on');

  let $order_emails_form_group = $('#MA_MERCHANT_ORDER_EMAILS').parents('.form-group');
  let $oos_emails_form_group = $('#MA_MERCHANT_OOS_EMAILS').parents('.form-group');
  let $return_slip_emails_form_group = $('#MA_RETURN_SLIP_EMAILS').parents('.form-group');

  // Bind change event
  $(document).on('change', $ma_merchant_order_radio, function(){
    $order_emails_form_group.toggle($ma_merchant_order_radio.find(':checked').val() == '1');
  });

  $(document).on('change', $ma_merchant_oos_radio, function(){
    $oos_emails_form_group.toggle($ma_merchant_oos_radio.find(':checked').val() == '1');
  });

  $(document).on('change', $ma_merchant_return_slip_radio, function(){
    $return_slip_emails_form_group.toggle($ma_merchant_return_slip_radio.find(':checked').val() == '1');
  });

  $ma_merchant_order_radio.trigger('change');
  $ma_merchant_oos_radio.trigger('change');
  $ma_merchant_return_slip_radio.trigger('change');
```